### ISSUE TITLE
Backbone 1.2.0 support

### ISSUE THREAD

**Issue by danwashusen** [tbranyen/backbone.layoutmanager#481]
Backbone 1.2.0 was released May 13th. 

I haven't tested if the current layoutmanager works with it yet so this is mostly a placeholder until more details are available.

**Comment by tbranyen** [comment]
Looks like it works fine in the browser, but fails in Node.  We relied on the fact that Backbone utilized jQuery for views to successfully pair them to the Cheerio library and make LM work in Node.  We can't support Backbone 1.2.0 officially because of that.

Just another reason to EOL Backbone.LayoutManager and rename to LayoutManager removing the Backbone reliance.

**Comment by Cleod9** [comment]
@tbranyen For clarification, do you have plans to continue this project yourself for Backbone 1.2.0 support?

**Comment by tbranyen** [comment]
@Cleod9 I dunno, I was probably exaggerating the amount of effort it'd take to support 1.2.0.  If you could look into it, that'd be really appreciated.

**Comment by tbranyen** [comment]
The major problem I can see is that since jQuery isn't used, there will be no easy way for Node parity through the use of Cheerio.

**Comment by Cleod9** [comment]
Node parity is one thing, but I was mainly just wondering about general client-side support for the library. I assumed the primary use case for this library would be client-side development.

**Comment by tbranyen** [comment]
@Cleod9 sure, and like i mentioned before the tests pass for the client side, but I can't merge anything into the dependencies manifest if it will break functionality that has existed for a long time.

**Comment by Cleod9** [comment]
Oh, ok that's understandable then. This is a really great library btw, to me it's like the "Backbone" of other Backbone extensions (doesn't do too much or too little). Would love to see continued support, though unfortunately I can't volunteer myself to assist with that right now :/

**Comment by Cleod9** [comment]
Good news everyone! I had some time to look into the problem, and I was able to fix the majority of the tests while using LM under Backbone 1.2.3. All I did was override the `_setElement` hook with one curated for cheerio:

https://github.com/tbranyen/backbone.layoutmanager/blob/master/node/index.js#L11
```
// Add cheerio hooks
Backbone.View.prototype._createElement = function(tagName) {
  var tag = "<" + tagName + ">" + "<" + tagName + "/>";
  return $.load(tag).root()[0].children[0];
};
```

There are only 4 more failing tests, all coming from this particular section:
https://github.com/tbranyen/backbone.layoutmanager/blob/master/test/spec/views.js#L1696

I'm not entirely sure what's wrong here, it might be something I'm missing with how cheerio works.

If anyone would like to assist with this that would be great, I'd love to see Layout Manager get back in the game!

References:
https://github.com/jashkenas/backbone/wiki/Using-Backbone-without-jQuery
https://gist.github.com/nhunzaker/9676031 (<-seems incorrect but helped)

**Comment by danwashusen** [comment]
@Cleod9 nice one :)

I'd be happy to sponsor development effort if that helps.

**Comment by Cleod9** [comment]
@danwashusen Would appreciate it! I won't have time to look at this again until next week at best so I could really use the help

**Comment by SimenB** [comment]
Any news here? Backbone is currently our only outdated dep, because of this project's dependency on an older version.

**Comment by michaJlS** [comment]
@Cleod9 

If you change location of `/` in the snippet which you've provided, so you have then:

```
Backbone.View.prototype._createElement = function(tagName) {
  var tag = "<" + tagName + ">" + "</" + tagName + ">";
  return $.load(tag).root()[0].children[0];
};
```

then it seems like all tests pass:

```
Running "nodequnit:files" (nodequnit) task
Testing node.js
...
>> 747 assertions passed (7423ms)

Done, without errors.

```

**Comment by Cleod9** [comment]
@michaJlS Whoa, so simple! I can't believe I didn't notice that silly mistake :sweat_smile: 

If that passes then well, does that mean we're done I suppose? I can get a pull request opened if there's nothing else outstanding.

**Comment by michaJlS** [comment]
@Cleod9 so maybe try to retest and see if I haven't missed anything.

**Comment by Cleod9** [comment]
`Done without errors` :wink:  

@tbranyen Created a PR just now: https://github.com/tbranyen/backbone.layoutmanager/pull/490

**Comment by tbranyen** [comment]
Landed #490
